samples/attestation/README.md

```diff
@@ -275,3 +275,16 @@ To use only the SGX remote functionality:
 * On Linux, use `make runsgxremote` instead of `make run`.
 
 * On Windows, use `ninja runsgxremote` instead of `ninja run`.
+
+## Determining Call Path for SGX Quote Generation
+
+Evidence of format `OE_FORMAT_UUID_SGX_ECDSA` can be generated by either taking the in-process call path or taking the out-of-process call path. However, if you wish to specify the call path it takes to generate a quote, here is what you can do:
+* To perform in-process quote generation, unset the environment variable `SGX_AESM_ADDR` and ensure that the DCAP library is installed.
+* To perform out-of-process quote generation, set the environment variable `SGX_AESM_ADDR` to any value and ensure that SGX SDK quote-ex Library is installed.
+
+On Linux, if `SGX_AESM_ADDR` is not set, one can run an existing OE app with out-of-process attestation, using `$ SGX_AESM_ADDR=1 <app_name>`.
```

Comment:
This is confusing because it only mentions Linux.  What about Windows?  Put the answer in the .md file.

Reply:
Added the answer to it. Thanks!